src/modules/75-cf/components/CreateEnvironmentDialog/EnvironmentDialog.tsx

```diff
@@ -150,100 +142,50 @@ const EnvironmentDialog: React.FC<EnvironmentDialogProps> = ({ disabled, onCreat
-                            itemFromNewTag={identity}
-                            tagInputProps={{
-                              showClearAllButton: true,
-                              allowNewTag: true,
-                              placeholder: 'Tags'
-                            }}
-                          />
-                        </Collapse>
-                      </Container>
-                    </Layout.Vertical>
-                    <Layout.Vertical spacing="small" style={{ margin: 'auto 0' }}>
-                      <Text font={{ size: 'normal' }} padding={{ top: 'medium' }}>
-                        {getEnvString('create.envTypeLabel')}
-                      </Text>
-                      <CardSelect
-                        cornerSelected
-                        data={envTypes}
-                        selected={getTypeOption(formikProps.values.type)}
-                        className={css.cardSelect}
-                        onChange={nextValue => formikProps.setFieldValue('type', nextValue.value)}
-                        renderItem={cardData => (
-                          <Container
-                            flex={{ align: 'center-center', distribution: 'space-between' }}
-                            className="cardBody"
-                          >
-                            {cardData.text}
-                          </Container>
-                        )}
-                      />
-                    </Layout.Vertical>
-                  </Layout.Vertical>
-                  <Container
-                    style={{
-                      display: 'flex',
-                      justifyContent: 'flex-start',
-                      alignItems: 'center',
-                      marginTop: 'auto'
-                    }}
-                  >
-                    <Button
-                      type="submit"
-                      text={getString('createSecretYAML.create')}
-                      intent="primary"
-                      disabled={loading}
+                          <Text font={{ variation: FontVariation.SMALL }}>{cardData.text}</Text>
+                        </Container>
+                      )}
                     />
-                    <Button text={getString('cancel')} onClick={() => formikProps.handleReset()} minimal />
-                    {loading && <Spinner size={16} />}
-                  </Container>
-                </Container>
+                  </Layout.Vertical>
+                </Layout.Vertical>
+                <Layout.Horizontal spacing="small" padding={{ top: 'xxlarge' }}>
+                  <Button
+                    variation={ButtonVariation.PRIMARY}
+                    type="submit"
+                    text={getString('createSecretYAML.create')}
+                    intent="primary"
+                    disabled={loading}
+                  />
+                  <Button variation={ButtonVariation.TERTIARY} text={getString('cancel')} type="reset" minimal />
```

Comment:
I think this aught to be secondary
```suggestion
                  <Button variation={ButtonVariation.SECONDARY} text={getString('cancel')} type="reset" minimal />
```

Reply:
yeah kinda confused about this bc in the [UX standardisation presentation](https://docs.google.com/presentation/d/1vsJr0whxRcRlGYqRUQiT9NMD_fD3ED41I-ufZcCcAxA/edit#slide=id.gfc601cb324_1_9), it say 'Cancel' buttons are tertiary

Reply:
@crystal-fong yeah, good point! `TERTIARY` is fine here 👍🏻